Modules/_blake2/impl/blake2-impl.h

```diff
@@ -140,6 +140,9 @@ static inline void secure_zero_memory(void *v, size_t n)
 {
 #if defined(_WIN32) || defined(WIN32)
   SecureZeroMemory(v, n);
+#elif deifned(__hpux)
```

Comment:
typo, should be "defined"

Reply:
good catch ...

Reply:
Here is my upstream fix: https://github.com/BLAKE2/libb2/commit/37062e55cc52895f5c79aeebe40f45409c008c90